src/main/java/com/jcabi/s3/Bucket.java

```diff
@@ -120,7 +120,10 @@ void remove(@NotNull(message = "S3 key can't be NULL") String key)
          */
         private final transient Bucket origin;
         /**
-         * Prefix.
+         * Prefix is a string which is used.
```

Comment:
In JavaDoc mentioned the test, which explains <code>Prefixed</code> class logic well, doesn't it?
Take a look at <code>com.jcabi.s3.mock.MkBucketTest#listsOckets()</code>